packages/progress-circle/src/ProgressCircle.ts

```diff
@@ -18,17 +18,24 @@ import {
     SpectrumElement,
     TemplateResult,
 } from '@spectrum-web-components/base';
-import { property } from '@spectrum-web-components/base/src/decorators.js';
+import {
+    property,
+    query,
+} from '@spectrum-web-components/base/src/decorators.js';
+import { ObserveSlotText } from '@spectrum-web-components/shared/src/observe-slot-text.js';
 import { ifDefined } from '@spectrum-web-components/base/src/directives.js';
 
 import progressCircleStyles from './progress-circle.css.js';
 
 /**
  * @element sp-progress-circle
  */
-export class ProgressCircle extends SizedMixin(SpectrumElement, {
-    validSizes: ['s', 'm', 'l'],
-}) {
+export class ProgressCircle extends SizedMixin(
+    ObserveSlotText(SpectrumElement, ''),
+    {
+        validSizes: ['s', 'm', 'l'],
```

Comment:
does this sizes doesnt come from the defaults? You might not need to explicitly specify the sizes here again